simplified-main/src/main/java/org/librarysimplified/main/MainActivity.kt

```diff
@@ -85,6 +86,21 @@ class MainActivity : AppCompatActivity(R.layout.main_host) {
   }
 
   private fun interceptDeepLink() {
```

Comment:
should method name reflect purpose of this Deep Link?

Reply:
The idea is that this intercepts all deep links, because otherwise there's a risk that different interceptor methods would all handle the same link.

But since this is the currently the only interceptor, this is a good idea.

And I don't want to completely delete the old `interceptDeepLink()` function, since we might want similar functionality in E-kirjasto, and upstream will have to rewrite it before too long (Firebase Dynamic Links are deprecated), so merging those changes will be more painful if we don't have it.

Reply:
I changed this so that `interceptDeepLink()` is the "main" handler (the only one that 's called), and then it decides whether to call `interceptDeepLinkTestLogin()` or `interceptDeepLinkBarcode()` (just a simple if-else for now, can be made smarter later, if other deep links are added).